renderdoc/driver/vulkan/vk_rendertexture.cpp

```diff
@@ -372,6 +372,8 @@ bool VulkanReplay::RenderTextureInternal(TextureDisplay cfg, const ImageState &i
       uint32_t texType;
       uint32_t selectedSliceFace;
       int32_t selectedSample;
+      float selectedRangeMin;
+      float selectedRangeMax;
```

Comment:
Unfortunately moving these from the end in here will break backwards compatibility :smile: - a custom shader could have been written that's using `YUVDownsampleRate` and `YUVAChannels`. Declaring the new variables on the end as you had it before should be fine as it's just "extra" data which isn't used, but in here this will mean the data is wrong in those variables for old shaders.

There's also a padding issue I think - the cbuffer has typical 16-byte alignment so this will push those `YUV` members further on and mismatch from the C struct here.